medusa/server/web/config/post_processing.py

```diff
@@ -213,19 +220,13 @@ def isNamingValid(pattern=None, multi=None, abd=False, sports=False, anime_type=
             return 'invalid'
 
     @staticmethod
-    def isRarSupported():
-        """
-        Test Packing Support:
-            - Simulating in memory rar extraction on test.rar file
-        """
-
+    def isRarSupported(unrar_tool):
```

Comment:
I completely disagree. I don't believe we aver chose to keep using bad practices, at the contrary, I'm sure we have chosen to start using best practices, we should definitely enforce them. Also, it has everything to do with this PR, as it directly changes code code that can and should be brought to our chosen standards.